instrumentation-genai/opentelemetry-instrumentation-vertexai/src/opentelemetry/instrumentation/vertexai/utils.py

```diff
@@ -106,16 +122,20 @@ def get_server_attributes(
 
 
 def get_genai_request_attributes(
+    use_latest_semconvs: bool,
     params: GenerateContentParams,
     operation_name: GenAIAttributes.GenAiOperationNameValues = GenAIAttributes.GenAiOperationNameValues.CHAT,
 ):
     model = _get_model_name(params.model)
     generation_config = params.generation_config
     attributes: dict[str, AttributeValue] = {
         GenAIAttributes.GEN_AI_OPERATION_NAME: operation_name.value,
-        GenAIAttributes.GEN_AI_SYSTEM: GenAIAttributes.GenAiSystemValues.VERTEX_AI.value,
         GenAIAttributes.GEN_AI_REQUEST_MODEL: model,
     }
+    if not use_latest_semconvs:
+        attributes[GenAIAttributes.GEN_AI_SYSTEM] = (
+            GenAIAttributes.GenAiSystemValues.VERTEX_AI.value
+        )
```

Comment:
Re:

> I don't like that new users have to know to set this env var and don't get all the newest stuff by default..
> 
> I realize what I've just described doing with an env var, is the same thing that we do with the standard release.. It's kind of like we want 2 releases one with breaking changes and one without..
> 
> Would be interested in others thoughts on this..

I don't love having to opt-in since what's under `if use_latest_semconvs` below is all additive. But I guess there is a case for it -- this one `GEN_AI_SYSTEM` attribute if not-latest. If it's really important to some users then it makes sense but I'm still quite new around genAI and instrumenting it to know.